app/lib/sanitize_config.rb

```diff
@@ -6,14 +6,14 @@ module Config
 
     CLASS_WHITELIST_TRANSFORMER = lambda do |env|
       node = env[:node]
-      class_list = node['class']&.split(' ')
+      class_list = node['class']&.split(/[\t\n\f\r ]/)
```

Comment:
wouldn't just `\s` work? according to docs, "/\s/ - A whitespace character: /[ \t\r\n\f\v]/"